smartapps/konnected-io/konnected-security-connect.src/konnected-security-connect.groovy

```diff
@@ -127,11 +126,7 @@ def pageDiscovery() {
   //create accessToken
   if(!state.accessToken) { createAccessToken() }
 
-  //This is a workaround to prevent page to refresh too fast.
-  if(!state.pageConfigurationRefresh) { state.pageConfigurationRefresh = 2 }
-
-  dynamicPage(name: "pageDiscovery", nextPage: "pageConfiguration", refreshInterval: state.pageConfigurationRefresh) {
-    state.pageConfigurationRefresh =  state.pageConfigurationRefresh + 3
```

Comment:
when is `initialize()` called?
In my testing, i was just going back and forth between the welcome page and Discovery page, and the refresh interval kept increasing. Maybe this is not a typical usage but still feels like a bug. Maybe instead we cap the refresh interval to say 10 seconds?